src/analytics/trackingEvents.ts

```diff
@@ -0,0 +1,25 @@
+// list of tracking events
+export enum generics {
+  pressedButton = 'Pressed Button',
```

Comment:
Future-looking (not a big deal rn), but we could encourage namespacing these as we migrate them to this new setup 🤔  i.e. `swaps` stuff always starts with `swaps`. This is usually something like high-level-to-low-level, left-to-right. So like `<category>_<feature>_<action>`. We'd also ideally align on action naming conventions, like past tense action followed by the target i.e. `pressedSwapButton`.

Reply:
agree with this, one of the reasons i'm leaning to using categories since we can then handle adding category/feature into automagically. 

from my reading its easier for tracking and filtering if we keep category/feature as properties of the action and this seems to be a well agreed upon standard